ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -183,11 +191,49 @@ - (BOOL)checkExtraQueryParameters
     return url!=nil;
 }
 
+- (ADAuthenticationError *)checkClaims
+{
```

Comment:
nit: While there’s nothing wrong in returning error like that, I would personally prefer returning boolean and error being a parameter. @RPangrle what do you think?